### PR TITLE
Dressing up some of the UI

### PR DESCRIPTION
**Pull request by JornC**
Some touchups to the user interface.

image

image

### CODE REVIEW

File: htmlstuff.php [hunk 1]
```diff
@@ -14,8 +14,10 @@ function pageheader() {
 
 function datadisclosure() {
 	global $userdata;
-	echo('This data will be saved with your poll results. If there is too much personal information included, please contact luke-jr to improve the filtering <em>before</em> filling out the poll.<br>');
-	echo('Note that if you have not completed KYC with Coinbase, your results will be saved but ignored until you complete KYC <em>and</em> resubmit your poll results.<br>');
+	echo('<div id="welcome">');
+	echo('<h1>Hello '.$userdata['coinbase_userdata']['data']['name'].'</h1>');
```
Comment (luke-jr) [inline htmlstuff.php:18]: This isn't really part of the disclosure. I suggest moving the <div.../h1> part to outside the functions (just in coinbase.php for now).

File: htmlstuff.php [hunk 2]
```diff
@@ -26,13 +28,27 @@ function formbegin() {
 
 function datadisclosure_checkbox() {
 	echo('<input id="accept_terms" name="accept_terms" type="checkbox" onclick="accept_terms_clicked()"' . (@$_POST['accept_terms'] ? ' checked' : '') . '><label for="accept_terms">I agree that the server may save this data, and that there is no promise of this data being kept secure</label><br>');
+
+	echo('</div>');
+	echo('<div class="polls">');
```
Comment (luke-jr) [inline htmlstuff.php:33]: Move this too

File: htmlstuff.php [hunk 3]
```diff
@@ -26,13 +28,27 @@ function formbegin() {
...
 	echo("</form>");
+	echo('</div>');
 	echo("<br><br>(the save button is on the top-right corner of your window)");
 }
 
 function pagefooter() {
 	echo("</body></html>");
 }
+
+function redirectToCoinbase() {
```
Comment (luke-jr) [inline htmlstuff.php:46]: This is only useful in index, so keep it there?

File: style.css [hunk 4]
```diff
@@ -1,3 +1,21 @@
+body {
+font-family: 'Roboto', sans-serif;
+display: flex;
+flex-direction: column;
+background-color: #d1ecff;
+align-items: center;
+}
+#welcome {
+    width: 80%;
+    max-width: 900px;
```
Comment (luke-jr) [inline style.css:10]: Won't using px not look sane on other displays? Why not use ex or em?

File: style.css [hunk 5]
```diff
@@ -10,15 +28,95 @@
...
+	background-color: #eef;
 }
 .pollsection th,td {
-	border: 1px solid black;
 	padding: 0.5ex 1ex;
 }
-.poll th {
-	text-align: left;
+th:first-child {
+	text-align: left !important;
```
Comment (luke-jr) [inline style.css:42]: Don't we want it centred for the options?

Comment (JornC) [reply style.css:42]: Yes, this rule applies only to the first child (the 'Do you agree with this' text), all the rest is centered.

File: style.css (hunk 4 above)
Comment (JornC) [reply style.css:10]: px is still pretty common for page widths and general layout afaik, mobile devices commonly scale these values with a pixel ratio based on pixel density on the screen with a reasonably sane default.

For example the Samsung Galaxy S4 has a full-HD resolution screen, and uses a CSS pixel ratio of 3 - making the effective size of that screen 360x640.

The defaults can be overriden, however they seem reasonable:

image

**Pull request merged**